Parse/src/main/java/com/parse/ParseObject.java

```diff
@@ -411,17 +402,7 @@ public ParseObject(String theClassName) {
    * @return A new {@code ParseObject} for the given class name.
    */
   public static ParseObject create(String className) {
-    if (objectTypes.containsKey(className)) {
-      try {
-        return objectTypes.get(className).newInstance();
-      } catch (Exception e) {
-        if (e instanceof RuntimeException) {
-          throw (RuntimeException) e;
-        }
-        throw new RuntimeException("Failed to create instance of subclass.", e);
-      }
-    }
-    return new ParseObject(className);
+    return getSubclassingController().instantiate(className);
```

Comment:
not sure how I feel about the name `instantiate(className)`. maybe `newInstance(className)` or `create(className)`?